include/dump_offload.hpp

```diff
@@ -289,6 +290,14 @@ inline void requestRoutes(App& app)
             handlers[&conn] = std::make_shared<Handler>(
                 *ioCon, dumpId, dumpType, unixSocketPath);
             handlers[&conn]->connection = &conn;
+
+            if (!crow::ibm_utils::createDirectory(unixSocketPathDir))
+            {
+                handlers[&conn]->connection->sendStreamErrorStatus(
+                    boost::beast::http::status::not_found);
```

Comment:
If we aren't able to create the directories, we should a not found?

Reply:
yeah. otherwise opening socket file will fail @gtmills